src/pages/billing/NewBillingProjectWizard/StepWizard/StepWizard.ts

```diff
@@ -0,0 +1,18 @@
+import { div, h2 } from 'react-hyperscript-helpers'
+
+
+export interface StepWizardProps {
+  title: string
+  intro: string
+  children?: React.ReactNode
```

Comment:
I don't think we ever what the StepHeader to have children. The text is always supposed to be below it.